FAQ.md

```diff
@@ -1,10 +1,14 @@
 # FAQ
 
 * What is Hacktoberfest?  (describe a short history, objectives, cool facts, etc)
+> Hacktoberfest is an initiative by Github and DigitalOceans ([website](https://hacktoberfest.digitalocean.com/)) and it's all about contributing to open source projects. After submitting 4 contributions you'll get a limited edtition Hacktoberfest 2017 T-Shirt. You can pick any project of your interest but all of contributions must be accepted within October.
+
 * Why are we doing this meetup? (so what's your motivation to do it)
+
 * Who could join?  (self explanatory I guess)
-* What's in it for me?
+> Anyone! It doesn't matter if you have years of experience with open source or you're just starting - We got your back. Developers, Designers, Translators - Open source needs you all.
 
-Fame and joy of showing off!
+* What's in it for me?
+> Fame and joy of showing off!
```

Comment:
i don't know about this line, it seems too hip for me 😂

Reply:
and showing off is not really the point IMO 🙂

Reply:
All this is intentional. Due to our target I made up a witty response and nobody will take it seriously ;)